test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_to_VR_DelCmd_WARNINGS_to_UI_DelCmd.lua

```diff
@@ -0,0 +1,167 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
```

Comment:
@istoimenova , script was updated in commit  d2baa99d69379db10009b64a0949b062a9e6933f